cf-custom-resources/lib/custom-domain-app-runner.js

```diff
@@ -364,9 +364,16 @@ async function updateCNAMERecordAndWait(recordName, recordValue, hostedZoneID, a
         HostedZoneId: hostedZoneID,
     };
 
-    const data = await appRoute53Client.changeResourceRecordSets(params).promise().catch((err) => {
+    let data;
+    try {
+        data = await appRoute53Client.changeResourceRecordSets(params).promise();
+    } catch (err) {
+        let recordSetNotFoundErrMessageRegex = /Tried to delete resource record set \[name='.*', type='CNAME'] but it was not found/;
+        if (action === "DELETE" && err.message.search(recordSetNotFoundErrMessageRegex) !== -1) {
```

Comment:
just curious, for my own learning: why -1 here?

Reply:
It returns `-1` if the regex is not found in the string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/search#return_value!

Reply:
Ahh, thanks!